src/clib/pio.h

```diff
@@ -595,7 +653,7 @@ extern "C" {
                             MPI_Comm io_comm, int *iosysidp);
     int PIOc_get_numiotasks(int iosysid, int *numiotasks);
     int PIOc_Init_Intracomm(MPI_Comm comp_comm, int num_iotasks, int stride, int base, int rearr,
-                            int *iosysidp);
+                            rearr_opt_t *rearr_opts, int *iosysidp);
```

Comment:
One issue however is that it would be harder (if the rearranger flow control options are explicitly listed) for users to ignore setting the rearranger options